src/sentry/workflow_engine/models/data_source.py

```diff
@@ -0,0 +1,23 @@
+from django.db import models
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import (
+    BoundedBigIntegerField,
+    DefaultFieldsModel,
+    FlexibleForeignKey,
+    region_silo_model,
+)
+
+
+@region_silo_model
+class DataSource(DefaultFieldsModel):
+    __relocation_scope__ = RelocationScope.Organization
+
+    class Type(models.IntegerChoices):
+        SNUBA_QUERY_SUBSCRIPTION = 1
```

Comment:
just wanted to double check this will grow?

Reply:
Yeah, we can just add more types here over time. I assume the idea is that this is an enum, and not against some dynamic db source?

Reply:
yep!